spec/features/embargo_spec.rb

```diff
@@ -0,0 +1,48 @@
+RSpec.feature 'embargo' do
+  let(:user) { create(:user) }
+  before do
+    sign_in user
+  end
+  describe 'creating an embargoed object' do
+    before do
+      visit '/'
+    end
+
+    let(:future_date) { 5.days.from_now }
+    let(:later_future_date) { 10.days.from_now }
+
+    # Pending revision to work in hyrax
+    xit 'can be created, displayed and updated', :workflow do
```

Comment:
Noted in https://github.com/samvera-labs/hyrax/issues/1123 and a new squashed commit will contain the comment link in the code.